DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_OverviewTable.cshtml

```diff
@@ -1,82 +1,82 @@
-﻿@using DigitalLearningSolutions.Data.Models.SelfAssessments;
+﻿@using DigitalLearningSolutions.Data.Models.SelfAssessments;
 @model IGrouping<string, Competency>;
-
-<table class="nhsuk-table-responsive nhsuk-u-margin-top-4">
-  <thead class="nhsuk-table__head">
-    <tr>
-      <th class="" scope="col">
-        @Model.First().Vocabulary
-      </th>
-      <th class="" scope="col">
-        @(string.IsNullOrWhiteSpace(Model.FirstOrDefault().QuestionLabel) ?
-                          "Question" : Model.FirstOrDefault().QuestionLabel)
-      </th>
-      <th class="" scope="col">
-        Self-assessment<br />status
-      </th>
-      @if (((CurrentSelfAssessment)ViewData["selfAssessment"])?.IsSupervisorResultsReviewed ?? true)
-      {
-        <th class="" scope="col">
-          Confirmation<br />status
-        </th>
-      }
-      <th class="" scope="col">
-        <span class="nhsuk-u-visually-hidden">Actions</span>
-      </th>
-    </tr>
-  </thead>
-
-  @foreach (var competency in Model)
-  {
-    <tbody class="nhsuk-table__body row-outer" id="comp-@competency.RowNo">
-      <tr class="nhsuk-table__row first-row">
+
+<table class="nhsuk-table-responsive nhsuk-u-margin-top-4">
+  <thead class="nhsuk-table__head">
+    <tr>
+      <th class="" scope="col">
+        @Model.First().Vocabulary
+      </th>
+      <th class="" scope="col">
+        @(string.IsNullOrWhiteSpace(Model.FirstOrDefault().QuestionLabel) ?
+                          "Question" : Model.FirstOrDefault().QuestionLabel)
+      </th>
+      <th class="" scope="col">
+        Self-assessment<br />status
+      </th>
+      @if (((CurrentSelfAssessment)ViewData["selfAssessment"])?.IsSupervisorResultsReviewed ?? true)
+      {
+        <th class="" scope="col">
+          Confirmation<br />status
+        </th>
+      }
+      <th class="" scope="col">
+        <span class="nhsuk-u-visually-hidden">Actions</span>
+      </th>
+    </tr>
+  </thead>
+
+  @foreach (var competency in Model)
+  {
+    <tbody class="nhsuk-table__body row-outer" id="comp-@competency.RowNo">
+      <tr class="nhsuk-table__row first-row">
         <td rowspan="@competency.AssessmentQuestions.Count()" class="nhsuk-table__cell">
+          <span class="nhsuk-table-responsive__heading">@competency.Vocabulary </span>

```

Comment:
I've moved this outside of the details expander and this seems to have fixed the alignment and made it consistent with the other items in the summary without CSS changes.